sys/nginx/default.conf.template.dev

```diff
@@ -24,6 +24,16 @@ server {
     listen 80;
     server_name poser.pugx.org poser.packagist.org poser.local stg.poser.pugx.org;
 
+    return https://$host;
```

Comment:
@JellyBellyDev Most likely https://github.com/PUGX/badge-poser/pull/1210#issuecomment-2371352585 it's caused by this.
The redirect is managed by nginx on port 443, but since it's exposed on the host on port 8001, the container is unaware of what is host port. I could change this to use `:8001` but it'll be coupling this template with the docker-compose.yaml, I think is kind of ok since only used for dev purposes, not really elegant, wydt?

Reply:
it's ok for me but it would be better to integrate this note here: https://github.com/PUGX/badge-poser/pull/1210/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R76  

what do you think?